tests/js-tests/project.json

```diff
@@ -1,3 +1,4 @@
+<<<<<<< HEAD
```

Comment:
I'm sorry @minggo.

In Github I see:

```
All checks have passed
1 successful check
@minggo Details Jenkins CI — Build successfully!
This branch has no conflicts with the base branch
Only those with write access to this repository can merge pull requests.
```

How resolve this new conflict?

Best and thanks!